examples/knapsack/optuna_example.py

```diff
@@ -155,20 +157,26 @@ def objective(trial: Trial):
     solver = solver_class(problem=problem, **kwargs)
     solver.init_model(**kwargs)
 
-    # solve
-    sol, fit = solver.solve(
-        callbacks=[
-            OptunaPruningSingleFitCallback(trial=trial, **kwargs),
-        ],
-        **kwargs,
-    ).get_best_solution_fit()
+    if True:
+        # solve
+        sol, fit = solver.solve(
+            callbacks=[
+                OptunaPruningSingleFitCallback(trial=trial, **kwargs),
+            ],
+            **kwargs,
+        ).get_best_solution_fit()
+    else:
+        raise optuna.TrialPruned()
 
     return fit
 
 
 # create study + database to store it
 storage = JournalStorage(JournalFileStorage(storage_path))
-optuna.delete_study(study_name=study_name, storage=storage)
+try:
```

Comment:
Sorry, i should have removed this line myself, it was to reset a previous study. As it is now it starts from scratch at each script run. If you remove or comment these lines, it optimizes further the study, remembering the previous trials.